guides/source/getting_started.md

```diff
@@ -113,6 +122,45 @@ $ sqlite3 --version
 
 The program should report its version.
 
+#### Installing Node
+
+Since Rails comes with Webpacker as JavaScript pre-processor and bundler,
+you will need to have Node runtime installed on your machine. Webpacker requires Node
+version 8.16.0 or later. Find the installation
+instructions at the [Node website](https://nodejs.org/en/download/) and 
+verify it's installed correctly with the following command:
+
+```bash 
+$ node --version
+```
+
+The version of your Node runtime should be printed out. Make sure it's greater
+than 8.16.0.
+
+You should also have NPM installed, so make sure this command prints out
```

Comment:
We don't need to mention npm if it's not used to install Yarn.

Reply:
Thanks!